crates/networking/rpc/engine/payload.rs

```diff
@@ -592,13 +594,6 @@ fn handle_new_payload_v3(
     block: Block,
     expected_blob_versioned_hashes: Vec<H256>,
 ) -> Result<PayloadStatus, RpcErr> {
-    // Ignore incoming
-    // Check sync status
-    match context.sync_status()? {
-        SyncStatus::Active | SyncStatus::Pending => return Ok(PayloadStatus::syncing()),
-        SyncStatus::Inactive => {}
-    }
-
```

Comment:
This will allow executing payloads on top of syncing state which will fail during snap sync